app/app.go

```diff
@@ -538,6 +542,11 @@ func (app *RegenApp) SimulationManager() *module.SimulationManager {
 	return app.sm
```

Comment:
do you still need this sim manager `sm *module.SimulationManager`?

Reply:
Yes, to initialize SDK modules simulations.

Reply:
So we're using the regular sdk `SimulationManager` for all except `WeightedOperation` which needs the new server module manager `smm`, right? Probably better to clarify that in some comments. We will likely need to revisit this as part of #301